tools/taskcluster/flank.yml

```diff
@@ -34,18 +33,16 @@ gcloud:
     - /sdcard/screenshots
   performance-metrics: true
 
-  # test-targets will be pulled dynamically from package.  DO NOT MODIFY
-  test-targets:
-TEST_TARGETS
-
   device:
-    - model: sailfish 
-      version: 26 
```

Comment:
Probably we'd also want num-flaky-test-attempts: 1, to repeat the test if it is failing intermittently, once we start running the tests in PRs

Reply:
Sounds good. Do you want me to update this in this PR? I hope we can land this soon and then iterate on the config as needed.

Reply:
Either way is fine by me. I can add it when I'm adding tests targets back

Reply:
Just saw that blueline is a physical device on Firebase's device list, and it might increase tests execution time by a lot, compared to a virtual device. Did you pick that for a reason or can I also change that, if it proves to be too slow?